package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Machine-to-machine interface for downloading form submissions",
   "repository": "git@github.com:cds-snc/forms-api.git",
   "scripts": {
-    "build": "tsc",
+    "build": "tsc -p tsconfig.build.json && tsc-alias -p tsconfig.build.json",
```

Comment:
Added `tsc-alias` package so that paths defined in `tsconfig.json`and used in the `.ts` files can be properly translated during compile time.